code/modules/jobs/job_types/roguetown/courtier/jester.dm

```diff
@@ -31,15 +31,17 @@
 	beltl = /obj/item/storage/belt/rogue/pouch
 	head = /obj/item/clothing/head/roguetown/jester
 	neck = /obj/item/clothing/neck/roguetown/coif
-	H.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
-	H.adjust_skillrank(/datum/skill/misc/reading, 4, TRUE)
-	H.adjust_skillrank(/datum/skill/misc/sneaking, 4, TRUE)
-	H.adjust_skillrank(/datum/skill/misc/stealing, 5, TRUE)
-	H.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE)
-	H.adjust_skillrank(/datum/skill/misc/athletics, 5, TRUE) //a showman like no other. you need to be fit to perform.
-	H.adjust_skillrank(/datum/skill/misc/music, rand(1,6), TRUE)
-	H.adjust_skillrank(/datum/skill/misc/lockpicking, 2, TRUE)
-	H.adjust_skillrank(/datum/skill/craft/cooking, rand(1,3), TRUE)
+	H.adjust_skillrank(/datum/skill/combat/knives, pick(3,4,5), TRUE)
+	H.adjust_skillrank(/datum/skill/misc/reading, pick(0,1), TRUE) // might just be an illiterate fuck, more likely than not
+	H.adjust_skillrank(/datum/skill/misc/sneaking, pick(1,2,3,4,4,4,5,6), TRUE)
+	H.adjust_skillrank(/datum/skill/misc/stealing, pick(1,2,3,4,4,4,5,6), TRUE)
+	H.adjust_skillrank(/datum/skill/misc/climbing, pick(4,5,6), TRUE)
+	H.adjust_skillrank(/datum/skill/misc/athletics, pick(4,5,6), TRUE) //a showman like no other. you need to be fit to perform.
+	H.adjust_skillrank(/datum/skill/misc/music, pick(1,4,4,6), TRUE)
```

Comment:
```suggestion
	H.adjust_skillrank(/datum/skill/misc/music, rand(1,6), TRUE)
```
remove weighting towards expert/legendary

Reply:
I'd argue this should be rand 4,6 honestly considering it's music